src/components/Button/Button.styles.ts

```diff
@@ -99,9 +98,8 @@ const getMemoizedBaseButtonStyles = memoizeFunction((props, theme): IButtonStyle
   };
 });
 
-export const getBaseButtonStyles = (props: BaseButtonStyleProps) => {
-  const theme = props.theme || getTheme();
```

Comment:
Good news is that this logic is built into `customizable`.  If were not wrapped in a `Customizer`, it will pull the global theme.

Reply:
Nice.